pkg/credentialprovider/plugin/plugin.go

```diff
@@ -325,6 +354,8 @@ type execPlugin struct {
 // The plugin is expected to receive the CredentialProviderRequest API via stdin from the kubelet and
 // return CredentialProviderResponse via stdout.
 func (e *execPlugin) ExecPlugin(ctx context.Context, image string) (*credentialproviderapi.CredentialProviderResponse, error) {
+	klog.V(5).InfoS("Getting image credentials from external exec plugin", "image", image, "plugin", e.name)
```

Comment:
The rest of this package doesn't use structured logging yet, so we can probably leave this as Infof and update it later with the rest of the package?

Reply:
Done!